### ISSUE TITLE
gulpESLintNew.fix() Error

### ISSUE THREAD

**Issue by Dulux-Oz** [origin-1/gulp-eslint-new#3]
I *think* there's an issue with gulpESLintNew.fix() - either that or my code is completely foobar (which, honestly, is way more likely) 😄 

Here's the error:
```
error TS2339: Property 'fix' does not exist on type '{ (options?: GulpESLintOptions | undefined):
ReadWriteStream; (overrideConfigFile?: string | undefined): ReadWriteStream; ... 5 more ...; format(formatter?: string
| ... 2 more ... | undefined, writer?: WritableStream | ... 1 more ... | undefined): ReadWriteStream; }'.
```
And here's the (relevant) code:
```
.pipe(gulpESLintNew({fix:true}))
.pipe(gulpESLintNew.format())
.pipe(gulpESLintNew.fix())
```

**Comment by fasttime** [comment]
Thanks for reporting. This has been fixed in version 1.4.2.

**Comment by Dulux-Oz** [comment]
Excellent! Thank you.
